docs/en/observability/monitor-gcp.asciidoc

```diff
@@ -0,0 +1,457 @@
+
+Next, click on *Create Service Account*. Then, define the new service account
+name (for example, "gcp-monitor") and the description (for example, "Service
+account to monitor {gcp} services using the Elastic Stack").
+
+image::monitor-gcp-service-account-name.png[Service account name]
+
+[IMPORTANT]
+====
+This next step is very important, so make sure to click on the correct roles.
+====
+
+In order to monitor {gcp} services you need to add two main roles to the
+service account:
+
+_Compute Viewer_:
+
+image::monitor-gcp-service-account-roles-compute-viewer.png[Service account roles compute viewer]
+
+_Monitoring Viewer_:
+
+image::monitor-gcp-service-account-roles-monitoring-viewer.png[Service account roles monitoring viewer]
+
+The final result should be the following:
+
+image::monitor-gcp-service-account-roles-final.png[Service account roles result]
+
+Click on *continue*, then skip granting users access to this service. Finally,
+click on *done* and the service account will be ready to be used.
+
+Next, in order to use the service account click on create keys and create the
+JSON key type.
+
+image::monitor-gcp-service-account-create-key.png[Service account create key]
+
+After clicking on create key, the credential file is downloaded. Keep this file
+in a accessible place to use later.
+
+[discrete]
+=== Step 2: Install and configure {metricbeat}
+
+In order to monitor {gcp} using the Elastic Stack you need two main components:
+an Elastic deployment to store and analyze the data and an agent to collect
+and ship the data.
+
+[NOTE]
+====
+This {tutorial} assumes the Elastic cluster is already running and ready
+to be used. Make sure you have your *cloudid* and your *credentials* in hand.
+====
+
+There are two agents that can be used to monitor {gcp}. {metricbeat} is used to
+monitor metrics and {filebeat} to monitor logs. You can run the agents in any
+machine. This {tutorial} uses a small {gcp} instance, e2-small (2 vCPUs,
+2 GB memory), with an Ubuntu distribution.
+
+[discrete]
+==== Install metricbeat
+
+After starting and connecting to the GCP instance, download and install
+{metricbeat}.
+
+include::{beats-repo-dir}/tab-widgets/install-widget-metricbeat.asciidoc[]
+
+[discrete]
+==== Set up assets
+
+{metricbeat} comes with predefined assets for parsing, indexing, and visualizing
+your data. Run the following command to load these assets.
+It may take a few minutes.
+
+[NOTE]
+=====
+Substitute the Cloud ID from your deployment in the following command. To find
+your Cloud ID Click on your deployment in https://cloud.elastic.co/deployments
+=====
+[source,bash]
+----
+./metricbeat setup -e -E 'cloud.id=YOUR_DEPLOYMENT_CLOUD_ID' -E 'cloud.auth=elastic:YOUR_SUPER_SECRET_PASS'
+----
+[IMPORTANT]
+====
+Setting up {metricbeat} is an admin-level task that requires extra privileges.
+As a best practice, https://www.elastic.co/guide/en/beats/metricbeat/7.11/privileges-to-setup-beats.html[use an administrator role to set up]
```

Comment:
We use [attributes](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc) to format cross-doc links. This enables link checking in our documentation build.

```suggestion
As a best practice, {metricbeat-ref}/privileges-to-setup-beats.html[use an administrator role to set up],
```